### DISCUSSION TITLE
Index multiple (but not all) subdirs with a single job

### DISCUSSION THREAD

**Discussion by kvvloten** [dadoonet/fscrawler#1688] [Q&A] [answered]
I would like to run a single job as a systemd service that indexes multiple subdirs in a tree like this (but not all of them):

```text
 /the/shared/root
    dir1
    dir2   <-- index everything in this dir
    dir3
    dir4/
       dir4.1
       dir4.2  <-- index everything in this dir
       dir4.3  <-- index everything in this dir
```
Is it possible to set `includes` and `excludes` such that the marked dirs are indexed and the others are ignored?

**Comment by reignblack** [comment, marked as answer]
Should be easy to do and its documented
https://fscrawler.readthedocs.io/en/latest/admin/fs/local-fs.html#includes-and-excludes
